linux_os/guide/system/accounts/accounts-restrictions/password_expiration/accounts_password_set_min_life_existing/tests/incorrect_min_pass_age.fail.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# platform = multi_platform_sle
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
```

Comment:
This doesn't have a meaning in the test scenarios, it's used in the bash remediation instead.

Reply:
Thank you @jan-cerny

Reply:
great, but please remove it from here